pubspec.yaml

```diff
@@ -43,18 +43,59 @@ flutter:
   assets:
     - fonts/google_fonts/
     - assets/demos/
-    - assets/studies/
     - assets/icons/cupertino/
     - assets/icons/material/
     - assets/icons/reference/
     - assets/icons/settings/
     - assets/logo/
-    - assets/splash_effects/
-    - packages/rally_assets/logo.png
-    - packages/rally_assets/thumb.png
-    - assets/crane/logo/
-    - assets/crane/destinations/
-    - assets/fortnightly/
+    - assets/studies/
+    - packages/flutter_gallery_assets/crane/destinations/eat_1.jpg
```

Comment:
It's annoying that when you use a package, you have to specify each asset individually. But oh well

Reply:
I'm working on a change to allow the same directory specifying as in local assets, just need more time to get it done :)